### PR TITLE
Hover over cells to see the row/col

### PR DESCRIPTION
**Pull request by darren-stoll**
I was having a lot of trouble trying to figure out which cell I was referencing when working on this project, so I added a 'title' to each cell in the sudoku grid so that when you hover over it it will display which cell you're on. It helped me a lot understand whether the letter referred to the column or row. 

An alternative solution would be to have x/y axes on the borders, but this is a quicker fix that I think would help.

### CODE REVIEW

**Comment by naomi-lgbt** [comment]
> An alternative solution would be to have x/y axes on the borders, but this is a quicker fix that I think would help.

I do think this is the better approach here, for a stronger UX and more accessibility. Would you be interested in updating this PR to reflect that? We can discuss the implementation further if you are interested.

**Comment by darren-stoll** [comment]
I'm a bit new to this, so forgive me for not understanding the process. I added another commit that addresses what you asked for.

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 Tested locally - I like the addition of the column/row axes. Additions do not impact the JavaScript that parses the puzzle string to the board.

Congratulations on your first pull request to this project! We look forward to future contributions!

Please note that our workflow requires at least one additional approval before this can be merged. 🙂

File: public/style.css
Comment (SaintPeter) [inline public/style.css:72]:
```suggestion
  text-align: center;
  vertical-align: bottom;
  padding-bottom: 0;
```

File: public/style.css
Comment (SaintPeter) [inline public/style.css]:
```suggestion
```

**Review by SaintPeter — DISMISSED** [review]
This looks pretty good, thank you for making the suggested changes.

I had a few minor styling tweaks.  I want to the axis labels to be closer to the grid.

On the side, remove the margin on the `#sudoku_grid` element, to get the left side axis closer to the grid.  On the top axis, remove the bottom padding and align to the bottom.

**Comment by naomi-lgbt** [comment]
Hey @darren-stoll,

Just checking in - have you had a chance to review Rex's suggestions above?

**Comment by darren-stoll** [comment]
Oops, I thought I already responded to this...

LGTM

**Comment by naomi-lgbt** [comment]
Would you mind committing them? You can do so by switching to file view, then selecting `Add suggestion to batch`, then after adding them you can `Commit Suggestions` at the top in one commit.

**Comment by darren-stoll** [comment]
I did it twice on accident, but I believe it's correct now?

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me. @SaintPeter would you mind taking another peek?

**Comment by SaintPeter** [comment]
Looks good to me!

**Pull request merged**
